core-models/src/main/java/com/anytypeio/anytype/core_models/ext/MembershipExt.kt

```diff
@@ -0,0 +1,38 @@
+package com.anytypeio.anytype.core_models.ext
+
+import com.anytypeio.anytype.core_models.membership.MembershipConstants
+import com.anytypeio.anytype.core_models.membership.MembershipUpgradeReason
+import com.anytypeio.anytype.core_models.membership.TierId
+
+fun TierId.isPossibleToUpgrade(
+    reason: MembershipUpgradeReason
+): Boolean {
+    return when (reason) {
+        MembershipUpgradeReason.NumberOfEditors -> isPossibleToUpgradeNumberOfSpaceMembers()
+        MembershipUpgradeReason.NumberOfReaders -> isPossibleToUpgradeNumberOfSpaceMembers()
+        MembershipUpgradeReason.NumberOfSharedSpaces -> false
```

Comment:
please add comment why it is false here.

Reply:
done